contrib/krb5-cmake/CMakeLists.txt

```diff
@@ -227,8 +224,8 @@ set(ALL_SRCS
     "${KRB5_SOURCE_DIR}/lib/crypto/openssl/sha256.c"
     "${KRB5_SOURCE_DIR}/lib/crypto/openssl/hmac.c"
     "${KRB5_SOURCE_DIR}/lib/crypto/openssl/pbkdf2.c"
-    "${KRB5_SOURCE_DIR}/lib/crypto/openssl/init.c"
-    "${KRB5_SOURCE_DIR}/lib/crypto/openssl/stubs.c"
+    "${KRB5_SOURCE_DIR}/lib/crypto/openssl/cmac.c"
```

Comment:
Cosmetic: The files listed in this CMakeLists.txt are ideally sorted to make the list easier to comprehend and to reduce merge conflicts. There has not been a consistent sorting before this PR (e.g. l. 224/225). But if you push a follow-up commit anyways, it would be great if you could apply sorting.